Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
+    system "python", "tools/setup.py"
+    system "python", "tools/build.py", "-C", "target/release"
+    ENV["DENO_BUILD_MODE"] = "release"
```

Comment:
Why are you setting a build variable after its been built ?

Reply:
This is required only for the test.py.